cmd/resource-generator/data/api.template

```diff
@@ -104,13 +104,14 @@ type {{ .Name }}Status struct {
 	// +patchStrategy=merge
 	// +listType=map
 	// +listMapKey=type
-	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+	// +optional
+	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
```

Comment:
We explicitly don't need, and shouldn't have, the protobuf stanza here. IIUC there's a flag to kal to disable this warning.

Reply:
OK, we let it slip https://github.com/k-orc/openstack-resource-controller/pull/172/files#diff-1f89fdaee723798b3f410ef995d7ccd352d8e71698447dd90bee022d467df5c6R90, for the record. I'll fix them both.

Reply:
It looks like we can only specify the flag when invoking kal as a linter. Lets ignore this one for now and just fix the others.

To be clear, though, we should *not* commit this change because it isn't correct.